### PR TITLE
Add rubygem-sidekiq{,-reliable-fetch}

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by ehelms** [comment]
You'll want to regenerate these using the scl gem2rpm template.

**Comment by adamruzicka** [comment]
Regenerated

**Comment by jlsherrill** [comment]
[test rpm]

File: packages/foreman/rubygem-gitlab-sidekiq-fetcher/rubygem-gitlab-sidekiq-fetcher.spec [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+%exclude %{gem_instdir}/.rspec
+%{gem_instdir}/Gemfile
+%{gem_instdir}/Gemfile.lock
+%doc %{gem_instdir}/README.md
+%doc %{gem_instdir}/RELEASE-GITLAB.md
+%{gem_instdir}/gitlab-sidekiq-fetcher.gemspec
+%{gem_instdir}/spec
+%{gem_instdir}/tests
+
+%changelog
```
Comment (jlsherrill) [inline packages/foreman/rubygem-gitlab-sidekiq-fetcher/rubygem-gitlab-sidekiq-fetcher.spec]: must add changelog entry!  (I wish the gem2rpm would do it fo us)

Comment (jlsherrill) [reply packages/foreman/rubygem-gitlab-sidekiq-fetcher/rubygem-gitlab-sidekiq-fetcher.spec]: same for the other

**Comment by jlsherrill** [comment]
you probably need to update comps/ and rel-eng/tito.props too!

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -327,11 +329,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
-      <packagereq type="default">nodejs-expose-loader-doc</packagereq>
-      <packagereq type="default">nodejs-file-loader-doc</packagereq>
       <packagereq type="default">nodejs-jquery-doc</packagereq>
-      <packagereq type="default">nodejs-jquery-flot-doc</packagereq>
-      <packagereq type="default">nodejs-select2-doc</packagereq>
```
Comment (adamruzicka) [inline comps/comps-foreman-rhel7.xml]: This was done by `comps_doc.sh`, should I put those back?

Comment (jlsherrill) [reply comps/comps-foreman-rhel7.xml]: ummmm, my guess is no, @ekohl?

**Comment by jlsherrill** [comment]
not sure what that error actually means...

[test rpm]

**Comment by jlsherrill** [comment]
@ekohl any idea about this failure?  It looks like the git annex whereis is failing, but it thinks the sources are already downloaded, i've never seen that before.

**Comment by adamruzicka** [comment]
And we're :green_apple:

File: comps/comps-foreman-rhel7.xml (hunk 2 above)
Comment (ekohl) [reply comps/comps-foreman-rhel7.xml]: We dropped various packages but probably didn't regenerate the comps file. Probably cleanest to fix this in `rpm/develop` and rebase this PR on it.

Comment (adamruzicka) [reply comps/comps-foreman-rhel7.xml]: Opened https://github.com/theforeman/foreman-packaging/pull/4195

**Comment by adamruzicka** [comment]
Is now based on https://github.com/theforeman/foreman-packaging/pull/4195

**Comment by ezr-ondrej** [comment]
@adamruzicka #4195 is in, could you rebase again?

File: packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+%license %{gem_instdir}/COMM-LICENSE
+%{gem_instdir}/Changes.md
+%{gem_instdir}/Ent-Changes.md
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/Pro-2.0-Upgrade.md
+%{gem_instdir}/Pro-3.0-Upgrade.md
+%{gem_instdir}/Pro-4.0-Upgrade.md
+%{gem_instdir}/Pro-Changes.md
+%{gem_instdir}/bin
+%{gem_instdir}/code_of_conduct.md
```
Comment (jlsherrill) [inline packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec]: The md files should probably have %doc on them, and the 'dot' files should probably have %exclude

File: packages/foreman/rubygem-gitlab-sidekiq-fetcher/rubygem-gitlab-sidekiq-fetcher.spec [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.gitlab-ci.yml
```
Comment (jlsherrill) [inline packages/foreman/rubygem-gitlab-sidekiq-fetcher/rubygem-gitlab-sidekiq-fetcher.spec]: %exclude these two dot files

File: packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec (hunk 3 above)
Comment (adamruzicka) [reply packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec]: Should all the md files go into the doc package?

Comment (ekohl) [reply packages/foreman/rubygem-sidekiq/rubygem-sidekiq.spec]: Probably. We should update our gem2rpm template since I've seen a few more of those.

**Pull request merged**